apps/web/global.d.ts

```diff
@@ -27,4 +27,14 @@ declare module 'next/config' {
 declare module '*.webm' {
   const src: string;
   export default src;
-}
\ No newline at end of file
+}
+
+declare module '*.glb' {
+  const src: string;
+  export default src;
+}
+
+declare module '*.gltf' {
+  const src: string;
+  export default src;
+}
```

Comment:
wat is this

Reply:
tells typescript not to complain when you import custom files.